www/pages/blog/release/v0-26-0.md

```diff
@@ -187,7 +187,7 @@ So although WCC is now the default for SSR, Puppeteer is still available as a pl
     
     export default {
       plugins: [
-        ...greenwoodPluginRendererPuppeteer()
+        greenwoodPluginRendererPuppeteer()
```

Comment:
I guess it makes sense to retro-actively fix this blog post?  Technically this _was_ the API at the time, should it stand as the part of the "record"? 🤔